src/components/site/Accordion.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const Accordion = props => {
+  const { canHaveMultiplePanelsOpen, className } = props;
+  const { collapseItems = [] } = props;
+
+  return (
+    <div
+      className={`dg_accordion ${
+        canHaveMultiplePanelsOpen === true ? "canHaveMultiplePanelsOpen" : ""
+      } ${className}`}
```

Comment:
this is a good case for the classnames package.

See [here](https://github.com/baltimorecounty/dotgov-components/blob/e90b2124ff91edd90e0d908732d035c5bfcce46e/src/components/basic/containers/Section.js#L7) for the implementation. 

In your case, for this, I think you can use this nifty trick.

```js
const cssClasses = classnames('dg_accordion', { 'canHaveMultiplePanelsOpen': canHaveMultiplePanelsOpen }, classname);
```

Reply:
This will prevent you from having the undefined class that is rendered now.

Reply:
https://www.npmjs.com/package/classnames

Reply:
this was originally added to handle the dg_section and dark. Accordion is no longer adding dg_section I've removed it entirely.